ruby/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'tzinfo'
```

Comment:
Yes.  Just running `bundler` pulls in all the referenced gems.
The caveat is you will probably need to run `bundler exec ./tzdump.rb`.
Hmm, I might add the bundler setup to the file...